src/test/isolation2/sql_isolation_testcase.py

```diff
@@ -581,6 +581,22 @@ def quit_process(self, out_file, name, mode="", dbname=""):
         self.processes[(name, mode)].quit()
         del self.processes[(name, mode)]
 
+    def terminate_process(self, out_file, name, mode="", dbname=""):
```

Comment:
parameter dbname is not used, do we really need it?

Reply:
It isn't used in quit_process either, the method I based terminate_process on. Should I remove it from both?

Reply:
I think this new code shall not have it.